packages/arco-lib/src/generated/types/Cascader.ts

```diff
@@ -3,13 +3,16 @@ import { StringUnion } from '../../sunmao-helper';
 import { Category } from '../../constants/category';
 import { CORE_VERSION, CoreWidgetName } from '@sunmao-ui/editor-sdk';
 
-const EXPRESSION_WIDGET_TYPE = `${CORE_VERSION}/${CoreWidgetName.Expression}`;
+const EXPRESSION_WIDGET_TYPE = `${CORE_VERSION}/${CoreWidgetName.Expression}` as const;
```

Comment:
It is duplicated in many files.
And Is it a must to add `as const`?

Reply:
Yes. If it doesn't have the `as const`, it would be the `string` type instead of `core/v1/expression` type.